src/interfaces/external/Convex/IFraxFarmERC20_V2.nc

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is this needed?

Reply:
I have removed Convex-Frax related work to a separate branch branched off of the `dev/macro-audit-14` branch.